pkg/web/static/js/main.js

```diff
@@ -43,4 +43,26 @@ document.body.addEventListener('htmx:responseError', (e) => {
       document.querySelector('.alert').remove()
     }, 5000)
   }
-})
\ No newline at end of file
+})
+
+/*
+ * Timestamp handling: localizes and formats datetimes on the page.
+ */
+const updateDatetimes = (e) => {
+  const options = {};
+  const dtfmt = Intl.DateTimeFormat(navigator.languages, {
+    year: 'numeric', month: 'long', day: 'numeric',
+    hour: 'numeric', minute:'numeric', second: 'numeric',
+    hour12: false, timeZoneName: "short",
+  });
+
+  const datetimes = document.querySelectorAll('.datetime');
+  datetimes?.forEach(elem => {
+    const dt = new Date(elem.textContent);
+    elem.textContent = dtfmt.format(dt);
+    elem.classList.remove('datetime');
```

Comment:
When a new resource is added (transaction, user, etc.), will the class name appear on the newly created element after the swap event?

Reply:
I should've noted that question was more for myself. I created an account and user and confirmed that the timestamp updated as expected.